### ISSUE TITLE
Useless scroll bar in the side panel

### ISSUE THREAD

**Issue by ezsh** [JJM152/Queen#179]
Falkon browser:
Application version 3.0.99 (8ffd2f20)
QtWebEngine version 5.12.1
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Falkon/3.0.99 Chrome/69.0.3497.128 Safari/537.36
![panel-scroll](https://user-images.githubusercontent.com/43062025/53693330-49dfaa00-3d9f-11e9-8494-7b72596a6e66.png)

**Comment by JJM152** [comment]
Does it go away if you turn off the inline numerical stats?

**Comment by ezsh** [comment]
No, it does not.

**Comment by JJM152** [comment]
It did for me after I reloaded the page. I'm trying to narrow down the cause.

**Comment by ezsh** [comment]
Reloading removes the avatar image and the scroll area extends a bit:
![panel-scroll-reload](https://user-images.githubusercontent.com/43062025/53693456-038b4a80-3da1-11e9-9c89-c65fb7758747.png)

**Comment by JJM152** [comment]
Testing a couple of other browsers, seems there is some differences in how it's interpreting the base sugarcube css and some overrides, gimme a few minutes.

**Comment by JJM152** [comment]
Seems like an issue with the overflow being set to auto on the ui-bar-body div. It previously had a calculated height using calc() but I shortened it to slip the avatar div underneath since I did not want to lose the auto refresh of the story elements in the other divs, but did not want the avatar to recalculate on every passage change.

Anyway, this is what I did:

- overflow: visible
- height: auto (to override sugarcube's behavior)
- extended the panel width when inline numbers are on, it was overflowing in chrome to the next line.

Seems to work fine for me now in chrome. If it's still broken in your browser then maybe it has an issue with setting the correct height on the div?
